libraries/swagger-validation-common/src/types/event.ts

```diff
@@ -0,0 +1,19 @@
+type Stage = "Queued" | "InProgress" | "Completed";
```

Comment:
Where use the stage

Reply:
Oh, sorry, this is not finished yet, I added the file by mistake, removing this in this pr

Reply:
@ruowan  removed, will sent out another pr including events